projects/packages/lazy-images/package.json

```diff
@@ -31,5 +31,9 @@
 	"engines": {
 		"node": "^12.18.0",
 		"yarn": "^1.3.2"
+	},
+	"com_jetpack": {
```

Comment:
I went with `com_jetpack` after reading https://stackoverflow.com/questions/10065564/add-custom-metadata-or-config-to-package-json-is-it-valid and the CommonJS package spec http://wiki.commonjs.org/wiki/Packages/1.1

I'm not tied to it if anyone has a different idea.